client/src/components/Register.js

```diff
@@ -4,17 +4,44 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { loadComponent } from 'lib/Injector'; // eslint-disable-line
 import availableMethodType from 'types/availableMethod';
+import registeredMethodType from 'types/registeredMethod';
 import LoadingIndicator from 'components/LoadingIndicator';
 
 class Register extends Component {
   constructor(props) {
     super(props);
+
+    const { registeredMethods, backupMethod } = props;
+
+    // Set initial selected method value based on props...
+    let selectedMethod = null;
+
+    // Set the backup method as the "selected" method if there are methods already registered for
+    // the user but one of those isn't the backup method.
+    if (
+      registeredMethods
+      && registeredMethods.length
+      && !registeredMethods.find(method => method.urlSegment === backupMethod.urlSegment)
```

Comment:
As discussed on slack, perhaps you could add `.length` to avoid any ambiguity around the comparison and the length check

Reply:
Yeah perhaps making it clear there's no registered method with `registeredMethods.filter(isABackupMethod).length === 0` is nicer

Reply:
I've updated that bit now :)